x/datapool/keeper/pool_test.go

```diff
@@ -304,7 +304,7 @@ func (suite poolTestSuite) TestBuyDataAccessNFTPending() {
 	suite.Require().Equal(pool.GetNumIssuedNfts(), uint64(1))
 }
 
-// TODO: TestBuyDataAccessNFTActive - check if data access NFT is mintes successfully
+// TODO: TestBuyDataAccessNFTActive - check if data pass is minted successfully
```

Comment:
In this test code, there are several 'data access NFT'. Please check them 🙏

Reply:
Oh I missed it 😅.

Fixed in [52846d6](https://github.com/medibloc/panacea-core/pull/325/commits/52846d6d05e825da4275f104132e1d7f7eb1d48b).